packages/bun-vscode/src/features/debug.ts

```diff
@@ -64,8 +74,15 @@ export function registerDebugger(context: vscode.ExtensionContext, factory?: vsc
       vscode.DebugConfigurationProviderTriggerKind.Dynamic,
     ),
     vscode.debug.registerDebugAdapterDescriptorFactory("bun", factory ?? new InlineDebugAdapterFactory()),
-    vscode.window.onDidOpenTerminal(injectDebugTerminal),
   );
+
+  if (getConfig("debugTerminal.enabled")) {
```

Comment:
Can we feature detect instead of relying on a particular version number? The installed extension may not be the VSCode version

Reply:
The version number is vscode and the stable release that included this new feature I use. I'm not 100% sure on if we can feature detect as the `ms-vscode.js-debug` extension has been around for a while (maybe we could just increase the package.json min version instead and not worry)

Reply:
this function is new right

Reply:
In versions before this was added, `jsDebugExt.exports` will be undefined, so feature detecting that should be pretty easy.

Reply:
oh ok thats awesome, thanks i'll update